src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -47,6 +50,26 @@ public abstract class CollectionItemsEqualConstraint : CollectionConstraint
         /// <param name="arg"></param>
         protected CollectionItemsEqualConstraint(object arg) : base(arg) { }
 
+        #region Protected Properties
+
+        /// <summary>
+        /// Get a flag indicating whether the user requested us to ignore case.
+        /// </summary>
+        protected bool IgnoringCase
+        {
+            get { return _comparer.IgnoreCase; }
+        }
+        
+        /// <summary>
+        /// Get a flag indicating whether any external comparers are in use.
+        /// </summary>
+        protected bool UsingExternalComparer
+        {
+            get { return _comparer.ExternalComparers.Count > 0; }
+        }
```

Comment:
How about `protected bool UsingExternalComparer => _comparer.ExternalComparers.Count > 0;`? Same for IgnoreCase?

Reply:
Just because C# 7 allows the syntax does not necessarily make it better.

Many, including myself, find it obscure and difficult to maintain.

Reply:
@oznetmaster C# 7 is not allowed in the codebase yet, but luckily I'd never make that insupportable argument about C# 6 either. ;-)

Reply:
I guess it is a matter of policy/preference.  None of my programmers are allowed to use that compact syntax because of clarity and maintainability.

Reply:
I agree Neil in not liking that syntax. Let's just leave it as a question of style.

Reply:
Yup! My reaction to the C# 1 version is pretty much the same as yours to the C# 6 one (minus the policy). 😊 Sorry Charlie, I think I remember now you saying you didn't like it, so no need to have brought it up.